### PR TITLE
docs: proposal about contributing file

### LINKED ISSUE #35: HOWTO: Contribute new components and use the build system
A publicly facing document on this would be really helpful.

### PR DESCRIPTION
**Pull request by Gregcop1**
How about helping people to report issues, propose pull request or writing component.  
This one should help to fix #35

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    * [Writing a pull request](#writing-a-pull-request)
+2. [Writing a component](#writing-a-component)
+    * [Component declaration](#component-declaration)
+    * [Building an example](#building-an-example)
+    * [Write some Tests](#write-some-tests)
+3. [Documentation](#documentation)
+
+## How to contribute
+
+First of all, if you read this, you may want to contribute, so thank you. It's a long and hard work and any help is welcome.
```
Comment (justindujardin) [inline CONTRIBUTING.md]: Try:
"If you are reading this you probably want to contribute, so thanks! Maintaining an open-source project is hard work and any help is welcome."

**Comment by justindujardin** [comment]
It's a really good start, thanks! I need to sort out the build failures before I can merge this which will probably be this weekend.  Thanks very much for putting this together!  

If you'd like to get it merged earlier, a PR that fixes the build would be welcomed--I think it's the rxjs dependency version not specifying exactly beta2 like angular wants.

**Comment by Gregcop1** [comment]
you mean, like [this one](https://github.com/justindujardin/ng2-material/pull/129)? ;)

**Comment by justindujardin** [comment]
@Gregcop1 great, thanks!

**Pull request merged**